packages/react-bootstrap-table2/src/contexts/index.js

```diff
@@ -280,6 +296,10 @@ const withContext = Base =>
         base = this.renderWithFilterCtx(base, baseProps);
       }
 
+      if (this.DragContext) {
```

Comment:
the context order is a important. please move to Line 274

The important thing here is the context order is about the whether a context is change raw data or not, so you see:  CellEdit, Filter, Search, Sort and Pagination, these context mush change raw data in different perspective.

Reply:
👍

Reply:
Is this going to be merged into the project?